### PR TITLE
[[ Bug 14866 ]] Make Xcode having a valid iOS SDK path, and make iOS SDK...

### PR DESCRIPTION
**Pull request by livecodesebastien**
... 8.2 the required one for iPhone deployment

### CODE REVIEW

**Comment by livecodesebastien** [comment]
Pull request https://github.com/runrev/livecode-ide/pull/114 contains the IDE changes for Xcode 6.2 / iOS 8.2 settings

File: ide-support/revdeploylibraryios.livecodescript [hunk 1]
```diff
@@ -352,6 +355,47 @@ command deployUpdateSDKs
...
+   -- PM-2015-02-06: [[ Bug 14422 ]] Removed OSX 10.6 and iOS 5.0 support
+   if revIDEDeployIOSSanatizeSystemVersion() < 100800 then
+      if "6.1" is not among the lines of it then
+         put false into tValid
+         put "In order to perfom device builds, you must have the iOS 6.1 SDK configured." & return & "The iOS 6.1 SDK comes with XCode 4.6." into tSDKStatus
+      end if
+   else if revIDEDeployIOSSanatizeSystemVersion() < 100900 then
+      if "7.1" is not among the lines of it then
+         put false into tValid
+         put "In order to perfom device builds, you must have either the iOS 7.1 SDK configured." & return & "The iOS 7.1 SDK comes with XCode 5.1." into tSDKStatus
```
Comment (runrevmark) [inline ide-support/revdeploylibraryios.livecodescript]: Should say 'must have the' (not must have either)

File: ide-support/revdeploylibraryios.livecodescript [hunk 2]
```diff
@@ -352,6 +355,47 @@ command deployUpdateSDKs
...
+      end if
+   else
+      -- SN-2015-03-12: [[ Bug 14866 ]] iOS SDK 8.2 is now the valid SDK
+      if "8.2" is not among the lines of it then
+         put false into tValid
+         put "In order to perfom device builds, you must have either the iOS 8.2 SDK configured." & return & "The iOS 8.2 SDK comes with XCode 6.2." into tSDKStatus
```
Comment (runrevmark) [inline ide-support/revdeploylibraryios.livecodescript]: Should say 'must have the'

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Pull request merged**